sdk/src/cameras/itof-camera/camera_itof.h

```diff
@@ -249,6 +252,10 @@ class CameraItof : public aditof::Camera {
     std::string m_initConfigFilePath;
     std::string m_ccbFile;
     aditof::ImagerType m_imagerType;
+
+    uint8_t *m_ab_video_data_8bit;
+    uint32_t m_abframeHeight;
+    uint32_t m_abframeWidth;
```

Comment:
This information can be obtained from a Frame object. There's no need to them here.